session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -506,11 +506,20 @@ public void createAlignedTimeseries(
       List<TSDataType> dataTypes,
       List<TSEncoding> encodings,
       List<CompressionType> compressors,
-      List<String> measurementAliasList)
+      List<String> measurementAliasList,
+      List<Map<String, String>> tagsList,
+      List<Map<String, String>> attributesList)
```

Comment:
IMO, adding a new interface rather than changing the old one is better for compatibility...

Reply:
Add don't forget update the doc of API.
https://github.com/apache/iotdb/blob/2f784502e52e15548babb2e4b9133f0a0731ba44/docs/UserGuide/API/Programming-Java-Native-API.md?plain=1#L150-L157

Reply:
ok